content/news/buzz/2017-google-summer-of-code-final-submission-adam-boniecki/contents.lr

```diff
@@ -0,0 +1,54 @@
+title: 2017 Google Summer of Code - Final Submission
+---
+author: Adam Boniecki
+---
+body:
+
+With Google Summer of Code 2017 program nearing its end, it is time to summarize what I got done during the summer working on `Batavia </project/projects/bridges/batavia/>`__.
+
+Batavia is a part of Pybee's BeeWare collection of projects. I posted my proposal in `this GitHub thread <https://github.com/pybee/batavia/issues/483>`__ and kept it updated with my progress on a weekly basis.
+
+Note that by the end of GSoC, we have decided to diverge from the proposal and forgo implementation of `contextlib` in favor of support for Python 3.6 2-byte wide opcodes.
```

Comment:
In the same vein as the above, it would great to have a sentence or two summary of what _was_ accomplished, and then the links below become supporting evidence to that summary.